app/src/views/Cart.vue

```diff
@@ -66,7 +66,7 @@
                 <figure class="aspect-w-16 aspect-h-9 shadow-light">
                   <img
                     class="w-full h-full object-center object-cover group-hover:opacity-90"
-                    :src="grantMetadata[metadataId(item.metaPtr)]?.logoURI || '/placeholder_grant.svg'"
+                    :src="ptrToURI(grantMetadata[metadataId(item.metaPtr)]?.logoPtr) || '/placeholder_grant.svg'"
                   />
```

Comment:
Not for now, but all of these images have a very similar structure (and the same list of classes), it might be worth abstracting these into a component.

Reply:
Added this to #265, good call